iv.yaml

```diff
@@ -0,0 +1,7 @@
+﻿lenguaje: node.js
+
+objeto valor:
```

Comment:
¿por qué? De verdad, ¿por qué volvéis a meter este error? ¿Dónde dice que tengáis que poner esto en el YAML?